packages/components/src/product-icon/index.jsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import paths from './paths';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+function ProductIcon( { className, slug } ) {
+	return (
+		<img
+			src={ paths[ slug ] }
```

Comment:
Should we render nothing instead of an empty `img` when we've specified an unsupported `slug`?

Reply:
Wouldn't the `PropTypes` like this:
```js
slug: PropTypes.oneOf( Object.keys( paths ) ).isRequired,
```
prevent the component from getting an unsupported `slug`?

Reply:
No. You'll get a warning that that slug is unexpected, but component will still get the unsupported props and render unexpected results.

Reply:
I'm going to add a check in this case 👍

Reply:
Could we not use slugs at all? Instead, export individual components for each product. That way, if a consumer only needs one of the products, they don't have to import everything and pay the cost of all packages. For consumers that want a slug-to-icon mapping, we could offer a factory function that does that separately.

Reply:
Hrm, though individual exports would make the main interface on `@automattic/components` a bit messy. `import { JetpackFreeProductIcon } from '@automattic/components'` feels a bit too verbose.

I think this is fine to land as-is. Maybe it makes more sense for this to live in its own package though? Releasing point releases to `@automattic/components` when we release new plan types feels odd.

Reply:
> Maybe it makes more sense for this to live in its own package though? Releasing point releases to @automattic/components when we release new plan types feels odd.

I don't see why it's odd TBH - we made a conscious decision to make this a reusable component and it makes a lot of sense in the a8c world, because we need it in multiple places. In that case, it only makes sense to release a new version when we add a new feature (icon) to a component (`ProductIcon`). 

If this sentiment is purely about point release vs major release, I agree, it would make sense to release a major release when including a new icon, and not prefer a point release.